scrapper.py

```diff
@@ -38,21 +40,73 @@ def slave(tasks, notifications, idx):
     
 
 def listener(addr, port):
+    #//TODO: Describe this function
     socket = zmq.Context().socket(zmq.REP)
     socket.bind(f"tcp://{addr}:{port}")
     
     while True:
         res = socket.recv()
         socket.send_json(True)
     
+
+def connectToSeeds1(sock, peerQ):
+    """
+    Thread that connect pull socket to seeds.
+    """
+    for addr, port in iter(peerQ.get, "STOP"):
+        with lockSocketPull:
+            log.debug(f"Connecting to seed {addr}:{port + 1}","Connect to Seeds1")
+            sock.connect(f"tcp://{addr}:{port + 1}")
+            counterSocketPull.release()
+            log.info(f"Scrapper connected to seed with address:{addr}:{port + 1})", "Connect to Seeds1")
+
+
+def disconnectFromSeeds1(sock, peerQ):
+    """
+    Thread that disconnect pull socket from seeds.
+    """
+    for addr, port in iter(peerQ.get, "STOP"):
+        with lockSocketPull:
+            log.debug(f"Disconnecting of seed {addr}:{port + 1}","Disconnect from Seeds1")
+            sock.disconnect(f"tcp://{addr}:{port + 1}")
+            counterSocketPull.acquire()
+            log.info(f"Dispatcher disconnected of seed with address:{addr}:{port + 1})", "Disconnect from Seeds1")
+
+
+def connectToSeeds2(sock, peerQ):
```

Comment:
use only a `connectToSeeds` with the appropriate parameters

Reply:
`connectToSeeds1` and `connectToSeeds2` are different in many ways, get notice. Logs, description, lock(remember that it is global) used and port used.

Reply:
both methods are exactly the same, you only need to pass some parameters